addons/popochiu/engine/objects/region/popochiu_region.gd

```diff
@@ -61,17 +63,23 @@ func _clear_scaling_region(chr: PopochiuCharacter) -> void:
 		chr.on_scaling_region = {}
 
 # ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ PRIVATE ░░░░
+func _check_scaling(area_rid: RID, area: Area2D, area_shape_index: int, local_shape_index: int, entered: bool):
+	if area is PopochiuCharacter and area.get_node("ScalingPolygon") and area_shape_index == area.get_node("ScalingPolygon").get_index():
```

Comment:
I'll just ask you to break the function parameters declaration and the first condition so they don't exceed column 100.

Also, sometimes Godot doesn't do well when comparing `get_node(...)`, so I prefer to use `get_node_or_null(...)`.

```suggestion
func _check_scaling(
	area_rid: RID, area: Area2D, area_shape_index: int, local_shape_index: int, entered: bool
):
	if (
		area is PopochiuCharacter and area.get_node_or_null("ScalingPolygon")
		and area_shape_index == $ScalingPolygon.get_index()
	):
```